.github/workflows/publish.yml

```diff
@@ -6,8 +6,17 @@ on:
     tags: ['*']
 
 jobs:
+  version-check:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check module version
+        uses: tarantool/actions/check-module-version@igrishnov/gh-204-add-hardcoded-versioning-support
+        with:
+          module-name: 'expirationd'
+
   publish-rockspec-scm-1:
     if: github.ref == 'refs/heads/master'
+    needs: version-check
```

Comment:
What result is expected for the master branch?

Reply:
What version should the master branch have if a latest tag is `1.2.3`?

Reply:
I don't quite understand the question, but the proposed procedure for manual versioning here is:

Let's say the latest release in the repository with the tag `1.2.3`

Let's say we are currently working on PR, after which we want to make a release with a new tag `1.2.4`. The last commit in this PR is a release commit, in which we describe the release and raise the version value in version.lua to `1.2.4`.

Next, the PR is merged into the master. 

After that, a release with a new tag `1.2.4` is made via the github web interface

Reply:
What if I don't need to change the version? Just some PR with a feature.

Reply:
Well, then we just don't touch the hardcoded value in the code